mne/inverse_sparse/mxne_inverse.py

```diff
@@ -433,10 +437,12 @@ def tf_mixed_norm(evoked, forward, noise_cov, alpha_space, alpha_time,
         Length of time window used to take care of edge artifacts in seconds.
         It can be one float or float if the values are different for left
         and right window length.
+    log_objective : bool
+        If True, the cost function is evaluated in every iteration.
     return_residual : bool
         If True, the residual is returned as an Evoked instance.
-    verbose : bool, str, int, or None
-        If not None, override default verbose level (see mne.verbose).
+    verbose: bool
+        Verbose output or not.
```

Comment:
-1, this goes against how `verbose` is used and described everywhere else in the package